pkg/statistics/hot_peer_cache.go

```diff
@@ -140,8 +142,8 @@ func (f *HotPeerCache) CollectExpiredItems(region *core.RegionInfo) []*HotPeerSt
 	regionID := region.GetID()
 	items := make([]*HotPeerStat, 0)
 	if ids, ok := f.storesOfRegion[regionID]; ok {
-		for storeID := range ids {
-			if region.GetStorePeer(storeID) == nil {
+		for storeID, isLeader := range ids {
```

Comment:
Will `justTransferLeader` be invalid?

Reply:
Made some changes, ptal again https://github.com/tikv/pd/pull/9372/commits/9e2197461dbe023cdd61601881de49e5b9858d81